packages/multiple-line/src/MultipleLineEditor.tsx

```diff
@@ -41,6 +55,17 @@ export function MultipleLineEditor(props: MultipleLineEditorProps) {
                 setValue(e.target.value);
               }}
             />
+            {withCharValidation && (
+              <div className={styles.validationRow}>
+                <CharCounter value={value || ''} checkConstraint={checkConstraint} />
+                <CharValidation constraints={constraints} />
+              </div>
+            )}
+            {!withCharValidation && (
+              <div className={styles.validationRow}>
+                <CharCounter value={value || ''} checkConstraint={() => true} />
```

Comment:
These wrapper divs need some a11y love, `aria-live="polite"` this will tell screen readers to announce the content when it changes 🙂 even better if we do it inside the CharCount component itself